common/src/main/java/tachyon/util/CommonUtils.java

```diff
@@ -64,6 +64,10 @@ public static void sleepMs(Logger logger, long timeMs, boolean shouldInterrupt)
     }
   }
 
+  public static void warmUpJVM() {
+    for (int k = 0; k < 10000000; k ++) {
+    }
+  }
```

Comment:
A minor format issue: please leave one extra line after this function before the javadoc of the next one.